src/components/renderBook.js

```diff
@@ -1,21 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import { removeBookFromApi } from '../redux/books/books';
 
 const RenderBook = ({ title, category, id }) => {
   const dispatch = useDispatch();
+
   const removeBookFromStore = (e) => {
     dispatch(removeBookFromApi(e.target.id));
   };
 
   return (
-    <li>
-      <p>{title}</p>
-      <p>{category}</p>
-      <button type="button" id={id} onClick={removeBookFromStore}>
-        remove
-      </button>
+    <li className="li-book">
+      <div className="book-info">
+        <p className="category">{category}</p>
+        <h3 className="book-title">{title}</h3>
+        <p className="author">Author unknown</p>
+        <div className="book-options">
+          <button className="btn-book" type="button">
+            Comment
+          </button>
+          <button
+            className="btn-book midle"
+            type="button"
+            id={id}
+            onClick={removeBookFromStore}
+          >
+            Remove
+          </button>
+          <button className="btn-book" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="book-circle">
+        <div className="circle">
+          <CircularProgressbar value={1} text="" />
+        </div>
+        <div>
+          <p className="percentage">1%</p>
+          <p className="completed">completed</p>
+        </div>
+      </div>
+      <div className="book-progress">
+        <p className="current-chapter">Current chapter</p>
+        <p className="chapter">introduction</p>
+        <button className="update-btn" type="button">
+          Update progress
+        </button>
+      </div>
```

Comment:
- [ ] According to the design, there should be a vertical separator between these two sections. please add it.

Reply:
Thanks for the review! I implemented the changes